Remove the padding of swiper wrapper

NRquadrat opened the pull request:
Padding no longer necessary. This provides for an incorrect calculation of the transition.

@hmerk Please test the improvment.

NRquadrat committed:
Remove the padding of swiper wrapper

Padding no longer necessary. This provides for an incorrect calculation of the transition.

hmerk commented on mainWidget/main_widget.yaml:
```diff
@@ -203,8 +202,8 @@ slots:
                           style:
-                            padding-left: 50px
-                            padding-right: 50px
+                            #padding-left: 50px
+                            #padding-right: 50px
```
Please remove those lines completely, not just add a hashtag

NRquadrat replied:
removed

hmerk commented on mainWidget/main_widget.yaml:
```diff
@@ -298,8 +297,8 @@ slots:
                             --swiper-navigation-size: var(--mW-topNavbar-swiper-navigation-size)
-                            padding-left: 50px
-                            padding-right: 50px
+                            #padding-left: 50px
+                            #padding-right: 50px
```
Same here, please remove

NRquadrat replied:
removed

hmerk commented on mainWidget/main_widget.yaml:
```diff
@@ -388,8 +387,8 @@ slots:
                             --swiper-navigation-size: var(--mW-topNavbar-swiper-navigation-size)
-                            padding-left: 50px
-                            padding-right: 50px
+                            #padding-left: 50px
+                            #padding-right: 50px
```
Same here, please remove

NRquadrat replied:
Removed

NRquadrat committed:
Remove Hashtags

NRquadrat committed:
various adjustments

- Reinstall --f7-button-text-transform: none; because of iOS compatibility
- Chip for floor display reinstalled, was forgotten

NRquadrat commented:
Hashtags removed,
Chip for Floor reinstalled
--f7-button-text-transform: none; reinstalled

hmerk reviewed (approved):
LGTM
